packages/flutter/lib/src/rendering/object.dart

```diff
@@ -760,6 +860,17 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
     markNeedsCompositingBitsUpdate();
   }
 
+  @override
+  void attach(Object newOwner) {
```

Comment:
I'll look into this, I didn't realize you could change signatures in an overridden method.